mongodb-atlas/src/main/java/io/vantiq/atlasConnector/AtlasStorageMgr.java

```diff
@@ -609,44 +618,52 @@ String buildStorageName(String databaseName, String collectionName) {
     public Completable
     startTransaction(String vantiqTransactionId, Map<String, Object> options) {
         return Completable.fromAction(() ->
-            transactions.asMap().computeIfAbsent(vantiqTransactionId, id -> {
-                Single<MongoClient> clientObs = connection.connect(config);
-                return clientObs.flatMapPublisher(MongoClient::startSession).doOnNext(ClientSession::startTransaction)
-                        .compose(new CacheIfNotEmptyOrError<>()).switchIfEmpty(
-                    // if we get here, startSession returned an empty publisher, should  never happen, but ...
-                    Flowable.error(new Exception("Failed to create a session in which to start a transaction"))
-                ).firstOrError().doOnSuccess(session ->
-                    log.debug("transactions started in session {} with number {}",
-                            session.getServerSession().getIdentifier(),
-                            session.getServerSession().getTransactionNumber())
-                );
-            })
+            transactions.asMap().computeIfAbsent(vantiqTransactionId, id -> 
```

Comment:
fixes a multitude of sins. old code only worked thanks to every request being in the same verticle